build.sbt

```
@@ -26,10 +26,10 @@ ThisBuild / githubWorkflowJavaVersions := Seq(
 )
 ThisBuild / githubWorkflowPublishTargetBranches := Seq.empty
 
-val Http4sVersion = "0.23.25"
-val CirceVersion = "0.14.6"
+val Http4sVersion = "0.23.26"
```

`0.23.27` has been dropped recently
```suggestion
val Http4sVersion = "0.23.27"
```